bridgetown-core/test/test_new_command.rb

```diff
@@ -60,7 +60,7 @@ def site_template_source
       static_template_files = dir_contents(site_template).reject do |f|
         File.extname(f) == ".erb"
       end
-      static_template_files << "/Gemfile"
+      static_template_files.push "/Gemfile", "/package.json"
```

Comment:
Definitely prefer your solution but fun fact:

```rb
static_template_files = []
=> []
static_template_files << "/Gemfile" << "/package.json"
=> ["/Gemfile", "/package.json"]
```

Reply:
Ah nice, cheers! Didn't know that was valid syntax :)